### PR TITLE
User assignee css issues fixed

### PR DESCRIPTION
**Pull request by sanbornsen**
- (fixes-#341)
- (fixes-#342)

### CODE REVIEW

File: src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.scss [hunk 1]
```diff
@@ -288,14 +289,15 @@
   .user-search {
     height: 100%;
     width: 100%;
-    position: relative;
+    position: absolute;
```
Comment (joshuawilson) [inline src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.scss]: I don't think this is the right way to do this.

File: src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.scss [hunk 2]
```diff
@@ -233,6 +233,7 @@
       .item-li {
         cursor: pointer;
         padding: 5px;
+        height: 30px; // Because images are 20 + padding 5. It helps in no name scenario
```
Comment (SMahil) [inline src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.scss]: remove height and add width:100% and float:left;

File: src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.scss [hunk 3]
```diff
@@ -288,14 +289,15 @@
...
-    position: relative;
+    position: absolute;
     z-index: 9;
     top: 0;
     left: 0;
     background-color: rgba(0, 0, 0, 0.6);
     .list-container {
       top: 12em;
       width: 80%;
+      position: relative;
```
Comment (SMahil) [inline src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.scss]: dont overwrite the position to relative and use transform to make it in center of div
we have mixin @include transform(translateX(-50%)); left:50%

File: src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.scss (hunk 1 above)
Comment (SMahil) [inline src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.scss]: use position fixed as this behaviour acts as popup

**Pull request merged**